dist/index.js

```diff
@@ -1811,7 +1812,8 @@ module.exports = class Processor {
 		}
 
 		const descriptionFileName = module.descriptionFileName.replace(/.md$/, '.html');
-		return fs.promises.readFile(`${this._contentPath}/${descriptionFileName}`);
+		const buffer = await fs.promises.readFile(`${this._contentPath}/${descriptionFileName}`);
+		return buffer.toString();
```

Comment:
just .toString?  Not .toString('utf-8')?

Reply:
I think `utf-8` is the default, but I'll fix this to be safe.